daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -330,9 +330,6 @@ abstract class AbstractFuncIT
       }
     }
 
-    // TODO https://github.com/digital-asset/daml/issues/10810
```

Comment:
wasn’t there a second test in the typescript bindings that we also disabled? cc @akrmn